packages/themes/scss/_theme-maps.scss

```diff
@@ -1,358 +1,11 @@
-  support-02: #42be65,
-  support-03: #f1c21b,
-  support-04: #4589ff,
-  inverse-support-01: #da1e28,
-  inverse-support-02: #24a148,
-  inverse-support-03: #f1c21b,
-  inverse-support-04: #0f62fe,
-  overlay-01: rgba(22, 22, 22, 0.7),
-  danger: #da1e28,
-  focus: #ffffff,
-  inverse-focus-ui: #0f62fe,
-  hover-primary: #0353e9,
-  active-primary: #002d9c,
-  hover-primary-text: #a6c8ff,
-  hover-secondary: #606060,
-  active-secondary: #393939,
-  hover-tertiary: #f4f4f4,
-  active-tertiary: #c6c6c6,
-  hover-ui: #353535,
-  active-ui: #525252,
-  selected-ui: #393939,
-  inverse-hover-ui: #e5e5e5,
-  hover-selected-ui: #4c4c4c,
-  hover-danger: #ba1b23,
-  active-danger: #750e13,
-  hover-row: #353535,
-  visited-link: #be95ff,
-  disabled-01: #262626,
-  disabled-02: #525252,
-  disabled-03: #6f6f6f,
-  highlight: #002d9c,
-  skeleton-01: #353535,
-  skeleton-02: #393939,
-  brand-01: #0f62fe,
-  brand-02: #6f6f6f,
-  brand-03: #ffffff,
-  active-01: #525252,
-  hover-field: #353535,
-) !default;
-/// Carbon's v9 color theme
-/// @type Map
-/// @access public
-/// @group @carbon/themes
-$carbon--theme--v9: (
-  interactive-01: #3d70b2,
-  interactive-02: #5a6872,
-  interactive-03: #5a6872,
-  interactive-04: #3d70b2,
-  ui-background: #f4f7fb,
-  ui-01: #ffffff,
-  ui-02: #f4f7fb,
-  ui-03: #dfe3e6,
-  ui-04: #8897a2,
-  ui-05: #5a6872,
-  text-01: #152935,
-  text-02: #5a6872,
-  text-03: #cdd1d4,
-  text-04: #ffffff,
-  text-05: #5a6872,
-  icon-01: #3d70b2,
-  icon-02: #5a6872,
-  icon-03: #ffffff,
-  link-01: #3d70b2,
-  inverse-link: #5596e6,
-  field-01: #ffffff,
-  field-02: #f4f7fb,
-  inverse-01: #ffffff,
-  inverse-02: #272d33,
-  support-01: #e0182d,
-  support-02: #5aa700,
-  support-03: #efc100,
-  support-04: #5aaafa,
-  inverse-support-01: #ff5050,
-  inverse-support-02: #8cd211,
-  inverse-support-03: #fdd600,
-  inverse-support-04: #5aaafa,
-  overlay-01: rgba(223, 227, 230, 0.5),
-  danger: #da1e28,
-  focus: #3d70b2,
-  inverse-focus-ui: #3d70b2,
-  hover-primary: #30588c,
-  active-primary: #30588c,
-  hover-primary-text: #294c86,
-  hover-secondary: #4d5b65,
-  active-secondary: #414f59,
-  hover-tertiary: #5a6872,
-  active-tertiary: #414f59,
-  hover-ui: #eef4fc,
-  active-ui: #dfeafa,
-  selected-ui: #eef4fc,
-  inverse-hover-ui: #4c4c4c,
-  hover-selected-ui: #dfeafa,
-  hover-danger: #c70014,
-  active-danger: #ad1625,
-  hover-row: #eef4fc,
-  visited-link: #294c86,
-  disabled-01: #fafbfd,
-  disabled-02: #dfe3e6,
-  disabled-03: #cdd1d4,
-  highlight: #f4f7fb,
-  skeleton-01: rgba(61, 112, 178, 0.1),
-  skeleton-02: rgba(61, 112, 178, 0.1),
-  brand-01: #3d70b2,
-  brand-02: #5a6872,
-  brand-03: #5a6872,
-  active-01: #dfeafa,
-  hover-field: #eef4fc,
-) !default;
-/// Carbon's default theme
-/// @type Map
-/// @access public
-/// @alias carbon--theme--white
-/// @group @carbon/themes
-$carbon--theme: $carbon--theme--white !default;
+
+// Deprecated import path
+// TODO: remove in next major release
```

Comment:
Maybe go with `// TODO: remove this file in next major release (v11)` or something in case we miss this deprecation and it then hangs around longer than expected?

Reply:
Down for whatever comment syntax we want to check for! No preference really